Content.Shared/Chat/MsgChatMessage.cs

```diff
@@ -54,6 +59,8 @@ public override void ReadFromBuffer(NetIncomingMessage buffer)
                     SenderEntity = buffer.ReadEntityUid();
                     break;
             }
+
+            MaxMessageLength = buffer.ReadInt16();
```

Comment:
It would probably be best if the net message for max length is split off into a new file, for example Content.Shared/Chat/MaxLengthMessage, that is sent to a new client whenever it joins and to all clients whenever the max message length is changed. Otherwise, the client would only know what the max length of the server is after someone sends a chat message and would be repeatedly told again every time someone does send one. That would leave MsgChatMessage as it was before.